components/Players/PlayerManager.vue

```diff
@@ -20,6 +20,11 @@
       <v-hover v-slot="{ hover }">
         <v-card class="player-card" width="100%">
           <video-player />
+          <up-next
+            v-if="showUpNext"
+            :time-left="timeLeft"
+            @hide="upNextUserHidden = true"
```

Comment:
Why not use
`v-is="showUpNext && !upNextUserHidden"`

Reply:
You would have to use $emit in `UpNext` component to tell the playerManager to hide

Reply:
`@hide` is an event emitted from the component which sets the `upNextUserHidden` variable to true, which basically means that the user want the component to be hidden until a new episode is shown. Might be a little confusing variable name tho, I'll try to thing of something better

Reply:
Haven't gotten any comments on this. Marking as resolved